eras/conway/impl/src/Cardano/Ledger/Conway/TxCert.hs

```diff
@@ -347,6 +387,14 @@ conwayTxCertDelegDecoder = \case
   15 -> do
     cred <- decCBOR
     pure (2, ResignCommitteeColdTxCert cred)
+  16 -> do
+    cred <- decCBOR
+    deposit <- decCBOR
+    pure (3, RegDRepTxCert cred deposit)
+  17 -> do
+    cred <- decCBOR
+    deposit <- decCBOR
+    pure (3, UnRegDRepTxCert cred deposit)
```

Comment:
We also need to adjust cddl spec for conway certificates. It's ok to add it in a subsequent PR too.

Reply:
Awesome catch! Hope I have done the right thing there now :crossed_fingers: